Sources/ExampleSwiftLibrary/MySwiftLibrary.swift

```diff
@@ -35,7 +35,7 @@ public func globalTakeIntInt(i: Int, j: Int) {
   p("i:\(i), j:\(j)")
 }
 
-public func globalCallMeRunnable(run: () -> ()) {
+public func globalCallMeRunnable(run: () -> Void) {
```

Comment:
c'mon, seriously swift-format? 🤪

(pretty sure that'll break some things since we checked the () -> () string 😅 )